src/pages/Product/components/RatingFilter/RatingFilter.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useFilter } from '../../../../context/filter/filterContext';
+
+function RatingFilter() {
+  const { filterState, filterDispatch } = useFilter();
+
+  return (
+    <>
+      <div className="filter-rating">
+        <h4>Rating</h4>
+        <div className="filter-rating-four">
+          <input
+            checked={filterState.rating === 4 ? true : false}
```

Comment:
You can just write (filterState.rating === 4) and this would also work because it would give true or false value only, you don't need to return it.